### DISCUSSION TITLE
build script for this repo ?

### DISCUSSION THREAD

**Discussion by nwmoon** [awslabs/aws-video-transcriber#44] [Q&A]
Hi 
after checking "video-transcriber-deploy.template",  i found the packages are in "aws-gcr-solutions/Video-Transcriber/v1.1.0/*"

and from the github, it seems every lambda scripts has its own build scripts, 

To contribute this source and build same packages as those in the repo, would anyone please share the build script and every package.json files for this repo ?

**Comment by zxkane** [comment]
Hi,

Please check if #45 to address your question
